src/plugin.scss

```diff
@@ -82,40 +88,21 @@
 
 .vjs-errors-content-container {
     overflow: auto;
-    position: absolute;
+    position: relative;
     padding-bottom: 15px;
-    top: 75px;
+    top: -10px;
```

Comment:
having negative values often means that we need to refactor some things because it isn't laid out correctly.
Some things to think about:
* why are we setting a negative value?
* any other "weird" styles being applied (`-3px` on the close button)
* how do you want things to look in the end?
  * is the dom layout set up to facilitate it?
  * what is the most minimal amount of css to get what we want?

It's possible that `-10px` is still what we want in the end here but it's time to step back for a minute and re-evaluate.